src/mapbox_logo.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var MapboxLogoControl = L.Control.extend({
+
+    options: {
+        position: 'bottomleft',
+    },
+
+    initialize: function(options) {
+        L.Util.setOptions(this, options);
+        this._mapboxLogo = '';
+    },
+
+    onAdd: function(map) {
+        this._container = L.DomUtil.create('div', 'mapbox-logo');
+        L.DomEvent.disableClickPropagation(this._container);
+
+        this._update();
+
+        return this._container;
+    },
+
+    addLogo: function(json) {
+        // If account referenced by the accessToken is
+        // asscociated with the Mapbox Logo, display logo.
+        var mapid = typeof this.options.urlOrMapID === 'string' && this.options.urlOrMapID.indexOf('/') === -1 ? true : false;
```

Comment:
Don't need the `? true : false`, here: `typeof this.options.urlOrMapID === 'string' && this.options.urlOrMapID.indexOf('/') === -1` is already a boolean value.